include/libcyphal/types.hpp

```diff
@@ -16,6 +18,19 @@
 namespace libcyphal
 {
 
+enum class Priority
```

Comment:
Transfer priority is also a transport-layer entity, so it should probably go there.